### ISSUE TITLE
New issue with rules and a specific macro.

### ISSUE THREAD

**Issue by Aalwein** [arkayenro/arkinventory#899]
Just recently I have begun having some issues with rules and order. I have a bunch of rules but the best example of my issue is this:

I have a rule set to identify "armor that is not quality 0 or 1", a rule set to identify armor that is part of my "tank outfit", and another rule that identifies armor that is part of my "fury outfit." When I first log into the game, the armor in the tank or fury outfit (whichever I'm not wearing) is properly sorted into the set inventory bar. But after switching out gear, suddenly the armor is no longer being identified as tank or fury but rather as armor, from the rule. I have the armor rule set to order 99, and the outfit rules set to 1, yet the armor rule (99) is overriding the outfit rules (1). I'm not sure why this is happening but I've never had an issue with it in the past. I also noticed if I disable the "armor" rule instead of setting the gear to fury or tank via rule, it simply stays at its default category (equippable items (soulbound)). I also noticed the rule doesn't get messed up until the spec actually changes (ie. I can swap gear back and forth with no problems until I actually change specs).

On further testing, I discovered the rule goes back into operation when I swap gear with the Blizzard gear changer. This issue is triggered when I use a macro to swap my spec and gear simultaneously. If it helps, the macro I use to swap gear is:

"/run if GetSpecialization()~= 2 then SetSpecialization(2); end"

TLDR; rule "armor" checks for any armor not quality 0 or 1 at order 99. rules "fury" and "tank" check for any armor in the named outfit at order 1. Rule "armor" (99) is taking precedence over "fury" and "tank" (1). Only occurs when using a specific macro to swap gear/spec simultaneously.
